packages/venia-ui/lib/components/Header/__stories__/header.js

```diff
@@ -17,7 +17,9 @@ stories.add('Search Bar Open', () => {
         const { toggleSearch } = api;
 
         // Open the search
-        toggleSearch();
+        useEffect(() => {
+            toggleSearch();
+        }, [toggleSearch]);
```

Comment:
Without the effect we had an infinite loop in this story. Not sure if changing out the provider is what caused this to start happening, but wrapping in an effect fixes it.

Reply:
Good catch. [Side effects definitely belong in `useEffect`](https://reactjs.org/docs/hooks-reference.html#useeffect).

> Mutations, subscriptions, timers, logging, and other side effects are not allowed inside the main body of a function component (referred to as React’s render phase). Doing so will lead to confusing bugs and inconsistencies in the UI.

We've seen with `console.log()` that it often works, but only coincidentally.